.github/workflows/java-gradle.yaml

```diff
@@ -1079,6 +1079,10 @@ on:
           KMS key name in GCP.
         required: false
 
+      BIN_REGISTRY_S3_BUCKET_URL:
```

Comment:
Isn't it better to just set environment variable on mixcr repo side?
(you may leave this solution, if you want to. I just suggest an option)

```yaml
secrets:
  env: |
    {
      "PL_REGISTRY_MILABORATORIES_UPLOAD_URL": ${{ toJSON(secrets.BIN_REGISTRY_S3_BUCKET_URL) }}
    }
```

Reply:
I just stick to the old ways of doing it. Thank you, I will consider your suggestion in future workflow development. :)